src/command/KeyBindingManager.js

```diff
@@ -410,6 +411,13 @@ define(function (require, exports, module) {
             });
         }
         
+        // Also make sure that the second word of PageUp/PageDown has the first letter in upper case.
+        if (/^Page/.test(key)) {
+            key = key.replace(/(up|down)$/, function (match, p1) {
+                return p1.charAt(0).toUpperCase() + p1.substr(1);
+            });
+        }
```

Comment:
@redmunds Good catch! This `if` statement is assuming that the other `if` block above it has been executed and all letters except the first one is already in lower case. So the culprit is in `/^[a-z]/.test(key)` where we should be testing with `i` flag.